cli/command/bundlefile/bundlefile.go

```diff
@@ -16,14 +16,15 @@ type Bundlefile struct {
 // Service is a service from a bundlefile
 type Service struct {
 	Image      string
-	Command    []string          `json:",omitempty"`
-	Args       []string          `json:",omitempty"`
-	Env        []string          `json:",omitempty"`
-	Labels     map[string]string `json:",omitempty"`
-	Ports      []Port            `json:",omitempty"`
-	WorkingDir *string           `json:",omitempty"`
-	User       *string           `json:",omitempty"`
-	Networks   []string          `json:",omitempty"`
+	Command    []string          			`json:",omitempty"`
+	Args       []string          			`json:",omitempty"`
+	Env        []string          			`json:",omitempty"`
+	Labels     map[string]string 			`json:",omitempty"`
+	Ports      []Port            			`json:",omitempty"`
+	WorkingDir *string           			`json:",omitempty"`
+	User       *string           			`json:",omitempty"`
+	Networks   []string          			`json:",omitempty"`
+	AutoRange  map[string]map[string]string `json:",omitempty"`
```

Comment:
Haven't looked at the feature as a whole, but we might want to skip updating the bundle file changes, as it never left the experimental stage, and is mainly obsoleted by the addition of stack deploy through compose-files

Reply:
Hi @thaJeztah,
In doubt i added it in the patch, didn't knew it was obsolete.
Will be removed in the next commit.